static/src/js/actions/focusedCollection.js

```diff
@@ -180,6 +180,7 @@ export const getFocusedCollection = () => async (dispatch, getState) => {
           items {
             conceptId
             definition
+            instanceInformation
```

Comment:
This change also needs to happen in actions/project.js, in the `GetProjectCollections` query. If you add the collection to your project, then visit the collection details page and refresh the page, you don't see the instanceInformation. Going directly to this link should show it missing

http://localhost:8080/search/granules/collection-details?p=C1200461634-CMR_ONLY!C1200461634-CMR_ONLY&pg[1][v]=t&pg[1][m]=download&ee=sit&q=C1200461634-CMR_ONLY&ac=true